sources/packages/api/src/cra-integration/cra-integration.service.spec.ts

```diff
@@ -44,6 +45,7 @@ describe("CRAIntegrationService", () => {
         givenName: faker.name.firstName(),
         birthDate: faker.date.past(18),
         taxYear: faker.random.number({ min: 2000, max: 2030 }),
+        userId: createFakeUser().id,
```

Comment:
can't we directly pass the object? maybe like `user: createFakeUser()`

Reply:
removed it, as changed the logic